master.css

```diff
@@ -350,7 +365,18 @@ label[for="sort"] {
     width: 260px;
 }
 .num-features {
-    position: absolute; top: -15px; right: 0px; font-size: 11px;
+    position: absolute;
+    top: -15px;
+    right: 0px;
+    font-size: 11px;
+    left: 0px;
+    background-color: #eee;
+    border-bottom: 1px solid #fff;
```

Comment:
The .num-features span was positioned absolutely above the th element, it didn't have a background, so the number of features would appear on top of the underlying table rows (after scrolling). I added a background color to the span and made it stretch the whole width of the th element